novuRajan/gannt-chart
# Ac2-gantt-130-adding-sub-task

**Description of the problem this PR solves**
Helps the user create the sub-task of the existing parent tasks.

**Details of how the solution has been implemented**
 Added button in the edit modal to add sub-task
Updated addTaskmodal and  addTask functions to handle the addition of sub-tasks as well
Ensuring unique IDs for new tasks and subtasks.

**Testing Instructions**
1. Right-click on the parent taskbar to open. the edit modal
2. Click the "Add Sub Task" button to add a new sub task.
3. Add the details and save changes
4. Confirm that the new sub-task is added below the selected parent task.
5. Now create a new task from the top left button.
6. Repeat steps 1 to 4 twice to create two new subtasks, 
7. Check the ID of the new tasks and sub-tasks to ensure it is unique and does not match the IDs of other existing tasks.
